[2022-06-05 09:53:57,701] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.controle_fluxo scheduled__2022-06-05T12:52:32.462045+00:00 [queued]>
[2022-06-05 09:53:57,711] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.controle_fluxo scheduled__2022-06-05T12:52:32.462045+00:00 [queued]>
[2022-06-05 09:53:57,711] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-05 09:53:57,711] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-05 09:53:57,712] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-05 09:53:57,726] {taskinstance.py:1378} INFO - Executing <Task(PythonOperator): controle_fluxo> on 2022-06-05 12:52:32.462045+00:00
[2022-06-05 09:53:57,734] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'controle_fluxo', 'scheduled__2022-06-05T12:52:32.462045+00:00', '--job-id', '821', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpz2rdv39i', '--error-file', '/tmp/tmpuc83u1sj']
[2022-06-05 09:53:57,732] {standard_task_runner.py:52} INFO - Started process 48271 to run task
[2022-06-05 09:53:57,734] {standard_task_runner.py:80} INFO - Job 821: Subtask controle_fluxo
[2022-06-05 09:53:57,808] {task_command.py:370} INFO - Running <TaskInstance: census.controle_fluxo scheduled__2022-06-05T12:52:32.462045+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-05 09:53:57,893] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=controle_fluxo
AIRFLOW_CTX_EXECUTION_DATE=2022-06-05T12:52:32.462045+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-05T12:52:32.462045+00:00
[2022-06-05 09:53:58,186] {python.py:173} INFO - Done. Returned value was: None
[2022-06-05 09:53:58,204] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=census, task_id=controle_fluxo, execution_date=20220605T125232, start_date=20220605T125357, end_date=20220605T125358
[2022-06-05 09:53:58,228] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-05 09:53:58,268] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
